### PR TITLE
New Telegram Desktop Cheat Sheet

### PR DESCRIPTION
**Pull request by TiagoDanin**
## Description of new Instant Answer, or changes

Add cheat sheet for Telegram Desktop:
- Keyboard Shortcuts.

## Related Issues and Discussions

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/telegram_desktop_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/telegram-desktop.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
+	"id": "telegram_desktop_cheat_sheet",
+	"name": "Telegram Desktop",
+	"description": "List of useful keyboard shortcuts for Telegram Desktop",
+	"metadata": {
+		"sourceName": "Telegram Desktop",
+		"sourceUrl": "https://github.com/telegramdesktop/tdesktop"
+	},
+ 	"aliases": [
+ 		"telegram", "telegramdesktop", "telegram desktop"
```
Comment (kirkins) [inline share/goodie/cheat_sheets/json/telegram-desktop.json]: You'll need to remove the 3rd alias as an alias can't be the same as the name.

File: share/goodie/cheat_sheets/json/telegram-desktop.json [hunk 2]
```diff
@@ -1,45 +1,52 @@
...
-	}
+    "id": "telegram_desktop_cheat_sheet",
+    "name": "Telegram Desktop",
+    "description": "List of useful keyboard shortcuts for Telegram Desktop",
+    "metadata": {
+        "sourceName": "Telegram Desktop",
+        "sourceUrl": "https://github.com/telegramdesktop/tdesktop"
+    },
+    "aliases": [
+        "telegram desktop"
```
Comment (kirkins) [inline share/goodie/cheat_sheets/json/telegram-desktop.json]: I meant you should remove the alias you kept. You can't have an alias which is the same as the name.

Comment (TiagoDanin) [inline share/goodie/cheat_sheets/json/telegram-desktop.json]: Oh, thank you. :)

File: share/goodie/cheat_sheets/json/telegram-desktop.json [hunk 3]
```diff
@@ -0,0 +1,52 @@
+{
+    "id": "telegram_desktop_cheat_sheet",
+    "name": "Telegram Desktop",
+    "description": "List of useful keyboard shortcuts for Telegram Desktop",
+    "metadata": {
+        "sourceName": "Telegram Desktop",
+        "sourceUrl": "https://github.com/telegramdesktop/tdesktop"
+    },
+    "aliases": [
+        "telegramdesktop"
```
Comment (sahildua2305) [inline share/goodie/cheat_sheets/json/telegram-desktop.json]: We could add more aliases here. A few possible aliases are - `desktop telegram` and `desktoptelegram`.

**Comment by TiagoDanin** [comment]
Anything missing?

**Comment by daxtheduck** [comment]
@TiagoDanin this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Pull request merged**